cogs/Voting.py

```diff
@@ -0,0 +1,644 @@
+#!/usr/bin/env python
+
+"""
+Koala Bot Vote Cog code and additional base cog functions
+Commented using reStructuredText (reST)
+"""
+# Built-in/Generic Imports
+from dotenv import load_dotenv
+from random import randint
+
+# Libs
+import discord
+from discord.ext import commands
+
+# Own modules
+import KoalaBot
+from utils import KoalaDBManager
+
+# Constants
+load_dotenv()
+
+
+class TwoWay(dict):
+    """Class to because a friend was bored and wanted a better way to make a two way dict than the way I had before"""
+    def __init__(self, dict_in=None):
+        """
+        Constructor method
+        :param dict_in: an existing dict to make two-way
+        """
+        super(TwoWay, self).__init__()
+        if dict_in is not None:
+            self.update(dict_in)
+
+    def __delitem__(self, key):
+        """
+        Remove an item from the dict
+        :param key: the key of the item
+        """
+        self.pop(self.pop(key))
+
+    def __setitem__(self, key, value):
+        """
+        Add an item to the dict. Errors if it already exists
+        :param key: the key of the item to add
+        :param value: the value of the item to add
+        """
+        assert key not in self or self[key] == value
+        super(TwoWay, self).__setitem__(key, value)
+        super(TwoWay, self).__setitem__(value, key)
+
+    def update(self, e, **f):
+        """
+        Update the dict
+        :param e: new dict to integrate into the existing one
+        :param f: keyword arguments
+        """
+        for key, value in e.items():
+            assert key not in self or self[key]==value
+            self[key] = value
+
+
+class Option:
+    def __init__(self, head, body, opt_id=None):
+        """
+        Object holding information about an option
+        :param head: the title of the option
+        :param body: the description of the option
+        """
+        if not opt_id:
+            self.id = randint(100000, 999999)
```

Comment:
Will likely need a larger number than this, go for int(18) [the same as discord ids].
Also a check to make sure this is not the same as another ID is needed, although this is unlikely, it is possible